codegen/src/lib.rs

```diff
@@ -176,7 +181,7 @@ pub fn arcdps_export(item: proc_macro::TokenStream) -> proc_macro::TokenStream {
 
             fn load() -> &'static ArcDpsExport {
                 let mut export = &EXPORT;
-                let res: Result<(), Box<dyn ::std::error::Error>> = #init;
+                let res: Result<(), Box<dyn ::std::error::Error>> = unsafe { #init };
```

Comment:
Why is the unsafe needed here?

Reply:
Thank you for looking at it!
It is indeed a breaking change and should only be added to a new version.

> Why is the unsafe needed here?

I was hoping you would catch that so I can get your recommendation on how we can avoid the:
**error[E0133]**: use of mutable static is unsafe and requires unsafe function or block
compiler error that arises when the unsafe block is removed.

I'm not good at all with macros and pretty new to Rust so I'm open to all suggestions!

Reply:
Thanks, if you're okay with it, I would push some changes, merge and then create a new release :)

Reply:
By all means. Thank you!